### ISSUE TITLE
Diablo 2 Resurrected black screen and VKD3D version

### ISSUE THREAD

**Issue by Conusoid** [lutris/lutris#3817]
I've been trying to run d2r through Lutris for quite some time. Usually from the very beginning I get a black screen with an obviously running game in the background, with music and a hand-cursor. This happening when using VKD3D version 2.4L-a19619.
If switch to VKD3D version 2.5L-db89d40 it hangs on black screen for about 5 sec and then crashing with standard Blizzard error.
Since if I run D2R through a blizzard client, an attempt to start the game does not display any logs, I use 'Run EXE inside Wine prefix' to start the game. Attaching the resulting logs in case they can help.

<details>
 <summary>Additional information</summary>
    
-     OS: Arch Linux x86_64
-     Kernel: 5.10.78-1-lts
-     CPU: Intel i5-4670 (4) @ 3.800GHz
-     GPU: NVIDIA GeForce GTX 770 (nvidia-470xx-dkms 470.86-2)
-     Memory: 15946MiB
-     Lutris 0.5.9.1
-     DXVK v1.9.2L-3e64e1b
-     lutris-ge-6.20-1-x86-64

</details>

[VKD3D_V2.4L-a19619c log (black screen, game running in the background)](https://gist.github.com/Conusoid/1336d5bcb780c46093db4ba995536f74)

[VKD3D_V2.5L-db89d40 log (5 sec black screen, crash)](https://gist.github.com/Conusoid/f2b876147062e0a9b9e8fb2ab20b42ac)

**Comment by cyhyraethz** [comment]
Have you tried following the instructions in [this forum post](https://forums.lutris.net/t/diablo-ii-resurrected-technical-preview-not-running/12471/3)?

I used those instructions to get the game working during the open beta, and it's been working great for me since then. The only issue I've experienced is missing character portraits in the lobby, but that is a very minor issue and otherwise the game has been running perfectly.

**Comment by GloriousEggroll** [comment]
I ran multiple tests and they all worked fine. Maybe try updating your drivers:
Test 1: 10400 + rtx 3070 495.44 driver on vkd3d db89d40
Test 2: 1185G + gt 1030 495.44 driver on vkd3d db89d40
Test 3: 5950x + 6900 XT mesa git driver on vkd3d db89d40

All of them were able to successfully go in game without issues. My guess is if something worked before and isnt now that maybe something was implemented that your current driver is not handling properly.

**Comment by Conusoid** [comment]
> Have you tried following the instructions in [this forum post](https://forums.lutris.net/t/diablo-ii-resurrected-technical-preview-not-running/12471/3)?

Yes, I tried all the solutions described in this post. In addition, as far as I understand, this fix has already been implemented in the lutris builds.

> All of them were able to successfully go in game without issues. My guess is if something worked before and isnt now that maybe something was implemented that your current driver is not handling properly.

This is exactly what bothering me! On other video cards (newer ones) everything works fine, but mine (although it falls into the list of minimum requirements) refuses. In addition, based on other forums, it is the owners of Kepler (GTX 600-700 series) video cards that experience similar issues.
The point is, I'm not saying that it's all lutris or wine fault. You guys are doing a great job and I am grateful to you that all these years I can enjoy games on my OS. I just wanted to point out that there's something changed in my case between 2 versions of the VKD3D and the resulting logs will be able to somehow identify an issue that does not allow the game to start normally.
As one of the contributors on [HansKristian-Work git](https://github.com/HansKristian-Work/vkd3d-proton/issues/818) pointed out, the problem is most likely in the drivers. So I just hope that there is at least some possibility that this can be corrected on our part, because there is absolutely no hope for nvidia's help.

**Comment by GloriousEggroll** [comment]
@Conusoid what I'm saying is your current drivers are outdated:

GPU: NVIDIA GeForce GTX 770 (nvidia-470xx-dkms 470.86-2) <----

And the issue may already be fixed in the newer nvidia 495.44 driver. Have you tried updating your driver version yet?

**Comment by Conusoid** [comment]
I would be very happy if this were the case, but they’re no longer offer driver support for its Kepler-based (GTX 600/700 series) graphics cards as of October 4, 2021. 
And it is not just words :D After installing 495 drivers, the xserver refuses to start with the  "the card is not supported by this driver" error. So I'm forced to use legacy driver from AUR. Moreover, the [nvidia site](https://imgur.com/a/3lVU9cx) also indicates my driver as the latest one for this card.
